internal/provider/provider.go

```diff
@@ -248,6 +248,13 @@ func azureProvider(supportLegacyTestSuite bool) *schema.Provider {
 				Description: "The OIDC ID token for use when authenticating as a Service Principal using OpenID Connect.",
 			},
 
+			"oidc_token_path": {
```

Comment:
would this be better as
```suggestion
			"oidc_token_file_path": {
```